WorldCat Discovery.js

```diff
@@ -0,0 +1,185 @@
+{
+	"translatorID": "159c65e6-329f-455e-9caa-e0c80bf8435e",
+	"label": "WorldCat Discovery",
+	"creator": "Andrew Shields",
+	"target": "^https?://[^/]+\\.on\\.worldcat\\.org",
+	"minVersion": "3.0.9",
+	"maxVersion": "",
+	"priority": 90,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcs",
+	"lastUpdated": "2015-01-09 21:18:25"
+}
+
+/*
+WorldCat Discovery translator for Zotero.
+Copyright (C) 2015 Andrew Shields
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+var detectWeb, doWeb, pageIsActionRecord, parseOpenURLData;
+
+detectWeb = function(doc, url) {
```

Comment:
Please follow the general conventions in other Zotero translators:
* Use `function name(args) {` to declare functions unless other methods make more sense. (there's a bit more reason for this than just convention)
* Declare and instantiate variables as needed, not towards the top of scope block.